src/Features/CSharp/Portable/EditAndContinue/BreakpointSpans.cs

```diff
@@ -801,6 +801,11 @@ internal static TextSpan GetEnvelope(SyntaxNode declaration)
                 return TextSpan.FromBounds(firstSpan.Start, lastSpan.End);
             }
 
+            if (declaration is CompilationUnitSyntax unit && unit.ContainsTopLevelStatements())
+            {
+                return TextSpan.FromBounds(unit.Members[0].SpanStart, unit.Members[^1].Span.End);
```

Comment:
- Is this different than `unit.SpanStart` and `unit.Span.End`?
- Either way, wouldn't the span contain any type declarations as well, not only the global statements? Is that intended? I'm noting that since the doc comment says `Returns a span that contains all possible breakpoint spans of top-level declaration and no breakpoint spans that do not belong to the declaration.`

Reply:
> Is this different than unit.SpanStart and unit.Span.End?

`unit.SpanStart` would include using statements.

But yes, this code needs updating though, for type declarations. See other comment.